packages/terra-base/src/Base.scss

```diff
@@ -1,6 +1,8 @@
-// Normalize box-sizing.
-// More info: https://css-tricks.com/box-sizing/#article-header-id-6
 html {
+  background-color: var(--terra-base-background-color, #fff);
+  background-image: var(--terra-base-background-image, none);
```

Comment:
Should we also include `background-repeat` within base?

Reply:
Yeah, that is a good idea. I just looked at the [consumer layout/nav](http://engineering.cerner.com/terra-consumer/iframe.html?selectedKind=Layout&selectedStory=Dex&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel), they use background image, size, position, and repeat in shorthand. I think it would be good to add those as well as. Let's go with the following to cover all use-cases we have now.

```
background-attachment: var(--terra-base-background-attachment);
background-color: var(--terra-base-background-color, #fff);
background-image: var(--terra-base-background-image);
background-position:  var(--terra-base-background-position);
background-repeat:  var(--terra-base-background-repeat);
background-size: var(--terra-base-background-size);
```

The CSS custom properties set without a fallback value will default to the browser default like we use now.